manifests/0000_50_cluster-monitoring-operator_02-role.yaml

```diff
@@ -156,340 +127,553 @@ rules:
   verbs:
-  - list
-  - watch
-- apiGroups:
-  - ''
-  resources:
-  - configmaps
-  - endpoints
-  - limitranges
-  - namespaces
-  - nodes
-  - persistentvolumeclaims
-  - persistentvolumes
-  - pods
-  - replicationcontrollers
-  - resourcequotas
-  - secrets
-  - services
-  verbs:
-  - list
-  - watch
-- apiGroups:
-  - coordination.k8s.io
-  resources:
-  - leases
-  verbs:
-  - list
-  - watch
-- apiGroups:
-  - certificates.k8s.io
-  resources:
-  - certificatesigningrequests
-  verbs:
-  - list
-  - watch
-- apiGroups:
-  - security.openshift.io
-  resourceNames:
-  - node-exporter
-  resources:
-  - securitycontextconstraints
-  verbs:
-  - use
-- apiGroups:
-  - user.openshift.io
-  resources:
-  - groups
-  verbs:
-  - list
-  - watch
-- apiGroups:
-  - build.openshift.io
-  resources:
-  - buildconfigs
-  - builds
-  verbs:
-  - list
-  - watch
-- apiGroups:
-  - route.openshift.io
-  resources:
-  - routes
-  verbs:
-  - list
-  - watch
-- apiGroups:
-  - apps.openshift.io
-  resources:
-  - deploymentconfigs
-  verbs:
-  - list
-  - watch
-- apiGroups:
-  - quota.openshift.io
-  resources:
-  - clusterresourcequotas
-  verbs:
-  - list
-  - watch
-- apiGroups:
-  - metrics.k8s.io
-  resources:
-  - nodes
-  - pods
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - metrics.k8s.io
-  resources:
-  - '*'
-  verbs:
-  - '*'
-- apiGroups:
-  - ''
-  resources:
-  - namespaces
-  - nodes
-  - pods
-  - services
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - ''
-  resources:
-  - nodes/metrics
-  verbs:
-  - get
-- nonResourceURLs:
-  - /metrics
-  verbs:
-  - get
-- apiGroups:
-  - ''
-  resources:
-  - configmaps
-  verbs:
-  - get
-- apiGroups:
-  - ''
-  resources:
-  - endpoints
-  - pods
-  - services
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - extensions
-  resources:
-  - ingresses
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - networking.k8s.io
-  resources:
-  - ingresses
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - monitoring.coreos.com
-  resources:
-  - alertmanagerconfigs
-  - alertmanagers
-  - alertmanagers/finalizers
-  - podmonitors
-  - probes
-  - prometheuses
-  - prometheuses/finalizers
-  - prometheusrules
-  - servicemonitors
-  - thanosrulers
-  - thanosrulers/finalizers
-  verbs:
-  - '*'
-- apiGroups:
-  - ''
-  resources:
-  - nodes
-  verbs:
-  - list
-  - watch
-- apiGroups:
-  - ''
-  resources:
-  - configmaps
-  - secrets
-  verbs:
-  - '*'
-- apiGroups:
-  - ''
-  resources:
-  - namespaces
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - apps
-  resources:
-  - statefulsets
-  verbs:
-  - '*'
-- apiGroups:
-  - ''
-  resources:
-  - pods
-  verbs:
-  - delete
-  - list
-- apiGroups:
-  - ''
-  resources:
-  - endpoints
-  - services
-  - services/finalizers
-  verbs:
   - create
-  - delete
   - get
   - update
-- apiGroups:
-  - monitoring.coreos.com
-  resources:
-  - alertmanagers
-  verbs:
-  - get
-
+  - delete
+- - apiGroups:
```

Comment:
Incorrect nesting level.

Reply:
Whoooops. 🙃

Reply:
seems like this is because of the for loop and because of `std.set([])` :shrug:

Reply:
Let's ditch the `std.set()` as it only complicates things here (sorry for this wrong suggestion).

With `std.set`:
```console
wc -l manifests/0000_50_cluster-monitoring-operator_02-role.yaml 
679 manifests/0000_50_cluster-monitoring-operator_02-role.yaml
```
Without `std.set`:
```console
wc -l manifests/0000_50_cluster-monitoring-operator_02-role.yaml 
679 manifests/0000_50_cluster-monitoring-operator_02-role.yaml
```

Reply:
For context, this is on `master`:
```console
wc -l manifests/0000_50_cluster-monitoring-operator_02-role.yaml 
495 manifests/0000_50_cluster-monitoring-operator_02-role.yaml
```